### ISSUE TITLE
panic: runtime error: index out of range [0] with length 0

### ISSUE THREAD

**Issue by networkop** [netrisai/netris-operator#43]
I'm installing netris-operator on a new cluster (without any calico resources) and seeing operator in `CrashLoopBackOff` with the following log:

```
{"level":"error","ts":1643045695.4067953,"logger":"LBWatcher","msg":"","error":"CRD Not found"}
I0124 17:34:55.406862       1 leaderelection.go:242] attempting to acquire leader lease  netris/abac3abe.netris.ai...
{"level":"info","ts":1643045695.4070582,"logger":"controller-runtime.manager","msg":"starting metrics server","path":"/metrics"}
panic: runtime error: index out of range [0] with length 0

goroutine 196 [running]:
github.com/netrisai/netris-operator/calicowatcher.(*Watcher).deleteProcess(0xc00029eb40, 0x16823fb, 0x15)
        /workspace/calicowatcher/main.go:319 +0xb29
github.com/netrisai/netris-operator/calicowatcher.(*Watcher).mainProcessing(0xc00029eb40, 0xc00004007b, 0x15)
        /workspace/calicowatcher/main.go:384 +0x1fd
github.com/netrisai/netris-operator/calicowatcher.(*Watcher).start(0xc00029eb40)
        /workspace/calicowatcher/main.go:137 +0x128
github.com/netrisai/netris-operator/calicowatcher.(*Watcher).Start(0xc00029eb40)
        /workspace/calicowatcher/main.go:159 +0x1e8
created by main.main
        /workspace/main.go:187 +0x1205
```

Steps to reproduce (assuming netris is already installed):

```
kubectl -n netris create secret generic netris-creds \
  --from-literal=host="http://10.110.50.235" \
  --from-literal=login="netris" --from-literal=password="newNet0ps"

helm install netris-operator netrisai/netris-operator \
--namespace netris

```

**Comment by networkop** [comment]
seems like this is the culprit https://github.com/netrisai/netris-operator/blob/b39bdd69423dd366bf2bdd5600df457af1e58a5b/calicowatcher/main.go#L319

**Comment by networkop** [comment]
everything works fine as long as I have at least one `BGPConfiguration` resource

**Comment by pogossian** [comment]
@networkop thanks for the report, should be resolved in #44

**Comment by networkop** [comment]
yep, lgtm. thanks @pogossian
